example/neumann-reduced/heat_equation_reduced.py

```diff
@@ -13,7 +14,7 @@
 
 from pymor_dealii.pymor.operator import DealIIMatrixOperator
 
-sys.path.insert(0, "../../lib")
+sys.path.insert(0, Path(__file__).resolve().parent.parent.parent / "lib")
```

Comment:
I see, I guess there is one parent too many here, though, which leads to additional failures. If the more recent python version is failing for the moment, that's no problem for me. Maybe it would be best to just keep it as a 'known to fail' test in order to remember this.

Reply:
Actually, no. That line isn't current.